README.md

```diff
@@ -47,32 +48,55 @@ Ensure you have the following installed:
 
 1. Clone the repository and install the required dependencies.
 
-   ```bash
-   git clone https://github.com/ONSdigital/dis-wagtail.git
-   ```
+    ```bash
+    git clone https://github.com/ONSdigital/dis-wagtail.git
+    ```
 
 2. Install dependencies
 
-   [Poetry](https://python-poetry.org/) is used to manage dependencies in this project. For more information, read
-   the [Poetry documentation](https://python-poetry.org/).
+    [Poetry](https://python-poetry.org/) is used to manage dependencies in this project. For more information, read
+    the [Poetry documentation](https://python-poetry.org/).
+
+    To install all dependencies, including development dependencies, run:
+
+    ```bash
+    make install-dev
+    ```
+
+    To install only production dependencies, run:
+
+    ```bash
+    make install
+    ```
+
+3. Build the docker container
+
+    ```bash
+    make docker-build
+    ```
 
-   To install all dependencies, including development dependencies, run:
+4. Run the docker container
 
-   ```bash
-   make install-dev
-   ```
+    ```bash
+    make docker-start
+    make docker-shell
```

Comment:
We agreed that the main way to run the project will be with Docker, so it is intentional.

It should work without too, but I'd rather direct folks to Docker to avoid issues with Python envs and dependencies

Reply:
Makes perfect sense to me - I remember raising the question but not sure I caught the result.

Reply:
updated to have a section with running it locally